src/styles/dynamic-page.scss

```diff
@@ -391,7 +391,7 @@ $block: #{$fd-namespace}-dynamic-page;
   }
 
   &--sm {
-    @include fd-responsive-page($fd-dynamic-page-header-padding-top-bottom, 0.5rem);
+    @include fd-responsive-page($fd-dynamic-page-header-padding-top-bottom, 1rem);
```

Comment:
But according to the Fiori specs `sm` paddings are 0.5rem

Reply:
Ooops, my bad, i've compared to the sapui5 where it's `1rem`, closing pr.